Lab1.py

```diff
@@ -0,0 +1,31 @@
+
+def Prime_series(number):
+  for iter in range(2,number):
+    if is_prime(iter) == True:
+      print(iter,end = " ")
+    else:
+      pass
+
+number = int(input("Enter the input Range : "))
+is_prime = lambda number: all( number%i != 0 for i in range(2, int(number**.5)+1) )
+Prime_series(number)
+
+def show_student(name, gpa=0):
+    print("Student name: "+name)
+    if gpa:
```

Comment:
No need for these lines if you assign a default value to the argument in the function definition `(name, GPA=0)`
It will take the default value in the definition if you do not pass it.